src/stores/game.js

```diff
@@ -366,10 +368,13 @@ export const useGameStore = defineStore('game', {
       }
     },
     makeSocketRequest(slug, data, method = 'get') {
-      const url =
-        import.meta.env.VITE_USE_GAMESTATE_API === 'true'
-          ? `/api/gamestate/${this.id}/move/${slug}`
-          : `/api/game/${slug}`;
+      const usingGameStateAPI = import.meta.env.VITE_USE_GAMESTATE_API === 'true';
```

Comment:
It's typically better to use the check by itself instead of assigning it to a variable. It aids in static analysis and tree shaking. I know we don't really do that in Cuttle but it's just good to consider longer term.

Reply:
Any sort of tool that statically analyses your code, if we ever do this in the client code it would also tree shake the variables so it’s best to standardize between them. Additionally I’m working on transpiling the server, or at least moving it to ES6 syntax so it’s beneficial to understand these things now even it’s not explicitly necessary yet.

You’re correct in that it’s fine as written now, but I would prefer we pick a style and stick to it between both client and server.